src/wallet/wallet.cpp

```diff
@@ -6753,13 +6718,15 @@ bool CWallet::ParameterInteraction(const CChainParams& params)
         if (nMaxFee > HIGH_MAX_TX_FEE)
             UIWarning(_("-maxtxfee is set to a very high fee rate! Fee rates this large could be paid on a single transaction."));
         maxTxFee = nMaxFee;
-        if (CFeeRate(maxTxFee, 1000) < ::minRelayTxFee)
+        if (maxTxFee < CalculateConventionalFee(LOW_LOGICAL_ACTIONS))
```

Comment:
The removed check was definitely an error (the node config wins out over the wallet config), while the new check should really just be a warning (if the user wants to constrain their maximum fee to be below `LOW_LOGICAL_ACTIONS`, that just constrains the kinds of transactions they can create; it does not prevent them from creating transactions). So we should keep the previous error check as-is, and instead introduce this new check as a `UIWarning`.

Reply:
Deferred to #6577 in the interest of unblocking concurrent PRs.